oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneIndexProviderService.java

```diff
@@ -104,201 +107,178 @@
 import static org.apache.jackrabbit.oak.spi.whiteboard.WhiteboardUtils.scheduleWithFixedDelay;
 
 @SuppressWarnings("UnusedDeclaration")
-@Component(metatype = true, label = "Apache Jackrabbit Oak LuceneIndexProvider")
+@Component(property = { "enableCopyOnReadSupport:Boolean=true", "enableCopyOnWriteSupport:Boolean=true" } )
```

Comment:
I would really recommend not mixing component property types with regular properties. Just put the private properties to the Configuration annotation type (but without the metatype annotations on the element). Also the default values only need to be defined once then (and do no longer require a dedicated constant) but can be added as literal on the component property type.

Reply:
Yes, I did that because otherwise the diff tool complains (the properties will not occur in the component.xml). Meanwhile @cziegele enlightened me. I'll fix it.